experimental/learn_descriptors/learn_descriptors.cc

```diff
@@ -0,0 +1,7 @@
+#include "learn_descriptors.hh"
+
+#include <iostream>
+
+void robot::experimental::learn_descriptors::hello_world(const std::string &in) {
```

Comment:
put the namespace on a separate line, so:
```suggestion
namespace robot::experimental::learn_descriptors {
void hello_world(const std::string &in) {
```

Reply:
don't forget the closing brace.

Reply:
done in commit 6d07ea92c4fc41e38ce5a3523b2719ab84e8a9dc